### ISSUE TITLE
Y axis label invisible on Dark Theme Mode. 

### ISSUE THREAD

**Issue by sigmarsson** [beto-rodriguez/LiveCharts2#1458]
**Describe the bug**
I change to dark theme and the Y axis labels aren't visible. X axis ones are. 

**To Reproduce**
```
        [DependsOn(nameof(Context))]
        public IEnumerable<ICartesianAxis> AY => new[]
        {
            new Axis
            {
                Name = Context?.Unit ?? "#",
                NameTextSize = 13,
                TextSize = 13,
                CrosshairLabelsBackground = SKColors.DarkOrange.AsLvcColor(),
                CrosshairLabelsPaint = new SolidColorPaint(SKColors.DarkRed, 1),
                CrosshairPaint = new SolidColorPaint(SKColors.DarkOrange, 1),
                CrosshairSnapEnabled = true
            }
        };
```

```
        <lvc:CartesianChart Series="{x:Bind ViewModel.Data, Mode=OneWay}"
                            XAxes="{x:Bind ViewModel.AX, Mode=OneWay}"
                            Sections="{x:Bind ViewModel.ThresholdSections, Mode=OneWay}"
                            YAxes="{x:Bind ViewModel.AY, Mode=OneWay}"
                            TooltipPosition="Hidden"/>
```

**Expected behavior**

**Screenshots**
<img width="136" alt="image" src="https://github.com/beto-rodriguez/LiveCharts2/assets/43340959/948ccede-19a9-4887-8756-8778e6c25f6e">

<img width="140" alt="image" src="https://github.com/beto-rodriguez/LiveCharts2/assets/43340959/6574108f-8a8b-4290-9d3d-135a5712adc3">

**Desktop (please complete the following information):**
 - OS: Microsoft Windows [Version 10.0.22631.3155]

**Smartphone (please complete the following information):**

**Additional context**

Nugets: 
```
<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.5.240205001-preview1" />
<PackageReference Include="LiveChartsCore.SkiaSharpView.Uno.WinUI" Version="2.0.0-rc2" />
```

**Comment by frogcrush** [comment]
In my case, I'm able to see the X and Y labels, but not the legend:

![image](https://github.com/beto-rodriguez/LiveCharts2/assets/6116333/739a7484-ade8-4ea4-9b02-4bfdbe8a0d01)

**Comment by sigmarsson** [comment]
Fabricate your own legend, it's better in the long term.
I.e. 
<Button.Flyout>
<Flyout>
...

**Comment by Bush-cat** [comment]
You can set the Design to Light/Dark Mode manually, I had to do that as well and simply did this:

```csharp
        // TODO only works with system theme! Integrate Theme Switcher
        if (App.Current.RequestedTheme == Microsoft.UI.Xaml.ApplicationTheme.Light)
        {
            LiveCharts.Configure(config => config.AddLightTheme());
        }
        else
        {
            LiveCharts.Configure(config => config.AddDarkTheme());
        }
```

**Comment by beto-rodriguez** [comment]
This is now fixed with #1662, things have changed a lot in the progress, this updates all the new visuals/features to use themes also.

This will be included in the next version of the library, thanks for the report!
